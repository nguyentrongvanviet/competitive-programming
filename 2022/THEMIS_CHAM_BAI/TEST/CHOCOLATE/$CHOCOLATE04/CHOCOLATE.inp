7 10 47 7 8
DLLLLDDRDRDDDRRLUULLUURURRUUDDDLDLDLDLDRRRRULUU