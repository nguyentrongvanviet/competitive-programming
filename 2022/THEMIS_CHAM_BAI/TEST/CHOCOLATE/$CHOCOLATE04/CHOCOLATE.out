9
1 1 1 2 3 3 4 6 49 